templates/repo/issue/view_content/comments.tmpl

```diff
@@ -329,7 +329,7 @@
 					<div class="detail">
 						{{svg "octicon-plus"}}
 						<span class="text grey muted-links">
-							<a href="{{.DependentIssue.HTMLURL}}">
+							<a class="muted" href="{{.DependentIssue.Link}}">
```

Comment:
Please check this line and below.

`class="muted"` is an unrelated change.

Reply:
Done.